### ISSUE TITLE
About: Check links in History 

### ISSUE THREAD

**Issue by Erin-Cecele** [thegetty/quire-docs#648]
### Before proceeding, check to make sure there isn’t an existing issue for your suggestion or a similar one.

- [X] I have searched the existing issues and determined this is new suggestion.

### Idea being proposed

Check links on this page. For example, the link to "An Editor's View of Digital Publishing" is out of date because Iris content moved. 

### What contribution category does your proposal fall under?

Copyedit text

**Comment by SilvestriStefano** [comment]
Hello,
I have clicked on each link on the current [About/History](https://quire.getty.edu/about/history/) page and found the following issues. I put the line numbers of the `history.md` file for reference:
- line 37:  [ARLIS/NA’s Multimedia & Technology Reviews](https://arlisna.org/publications/multimedia-technology-reviews/1025-roman-mosaics-in-the-j-paul-getty-museum)  returns a 404. I think I have found the article though [https://works.hcommons.org/records/xf3nn-fej08](https://works.hcommons.org/records/xf3nn-fej08) 

![Image](https://github.com/user-attachments/assets/9eca3205-22a3-4381-afd2-f720be793ab5)

- line 65: [A major release](https://github.com/gettypubs/quire-starter-theme/releases/tag/v0.1.0.alpha.15.3) returns a 404. Maybe remove it since the repository no longer exists? Or maybe it should link to Ramon Tapales repo [https://github.com/rtapales/quire-starter-theme](https://github.com/rtapales/quire-starter-theme) 

![Image](https://github.com/user-attachments/assets/a207a647-de8c-4f64-95aa-b8408819711e)

- line 81: [Open Publishing Fest](https://www.openpublishingfest.org/) is completely dead. I would suggest to change it to either their [gitlab page](https://gitlab.coko.foundation/open-publishing/openpublishingfest) or the [Coko Foundation announcement](https://coko.foundation/blog/announcing-the-open-publishing-fest-june-3-14.html) of it.

![Image](https://github.com/user-attachments/assets/5997400e-05e7-4fdd-bb93-f963a27df1b8)

- line 91: [newsletter](https://newsletters.getty.edu/h/t/DDE7B9372AAF01E4) points to a blank page , idk if it is intentional. This is also the same link used in the footer.

![Image](https://github.com/user-attachments/assets/25a91709-9a56-4ecb-a974-a97f044fb562)

>The link to the article [An Editor's View of Digital Publishing](http://blogs.getty.edu/iris/an-editors-view-of-digital-publishing/) mentioned in your post is not broken, but that you probably already know...

**Comment by SilvestriStefano** [comment]
> Just a note that the newsletter link and the Getty _Iris_ article are working fine for me. Thank you again for these excellent and detailed contributions!
Yes the _Iris_ article was working fine for me as well, that's why I said it is not broken 😸 . You are right the newsletter link is working, the AdBlocker Ultimate extension on my browser was blocking the content.

Glad I could help. I really enjoy what you all have created. Hopefully I could contribute more in the future.

**Comment by Erin-Cecele** [comment]
We absolutely welcome more contributions @SilvestriStefano. Thank you for being a meaningful member of this community.
